README.md

```diff
@@ -171,11 +190,10 @@ There are some good docs from the [NodeJS team](https://nodejs.org/api/packages.
   "exports": {
     ".": {
       "types": "index.d.ts",
-      "script": "index.umd.js",
+      "browser": "index.umd.js",
```

Comment:
I was initially a little hesitant about the browser field, due to some misunderstandings I had about how Vite currently handles it. However, I went through and re-read the latest PR again and realized I had misunderstood. They'll look first for browser, but then determine what kind of module the browser field points to; if it's UMD, then they use the module field instead. https://github.com/vitejs/vite/pull/9653